libbeat/docs/shared-autodiscover.asciidoc

```diff
@@ -203,39 +203,173 @@ Example:
   Different Beats that refer to the same leader lease will be competitors in holding the lease
   and only one will be elected as leader each time.
 
-The configuration of templates and conditions is similar to that of the Docker provider. Configuration templates can
-contain variables from the autodiscover event. They can be accessed under data namespace.
+Configuration templates can contain variables from the autodiscover event. These variables can be accessed under the `data`
+namespace, e.g. to access Pod IP: `${data.kubernetes.pod.ip}`.
 
-These are the fields available within config templating. The `kubernetes.*` fields will be available on each emitted event.
+These are the fields available within config templating. The `kubernetes.*` fields will be available on each emitted event:
 
 [float]
 ====== Generic fields:
   * host
   * port (if exposed)
-  * kubernetes.labels
-  * kubernetes.annotations
 
 [float]
 ====== Pod specific:
-  * kubernetes.container.id
-  * kubernetes.container.image
-  * kubernetes.container.name
-  * kubernetes.namespace
-  * kubernetes.node.name
-  * kubernetes.pod.name
-  * kubernetes.pod.uid
+|===
+|Key |Type |Description
+
+|`kubernetes.namespace`
+|`string`
+|Namespace, where the Pod is running
+
+|`kubernetes.namespace_uuid`
+|`string`
+|UUID of the Namespace, where the Pod is running
+
+|`kubernetes.namespace_labels.*`
+|`object`
+|Labels of the Namespace, where the Pod is running. All namespace labels are available by default, see `add_resource_metadata` configuration setting above.
+
+|`kubernetes.namespace_annotations.*`
+|`object`
+|Annotations of the Namespace, where the Pod is running. Namespace annotations are not available by default, see `add_resource_metadata` configuration setting above.
```

Comment:
This is not accurate as far as I see. `kubernetes.namespace_annotations`, `kubernetes.annotations` and `kubernetes.labels` are by default available for conditional templating regardless of include, exclude settings.  Also they are not dedoted regardless of the settings.
See https://github.com/elastic/beats/blob/fffe228735005439f684e4bc28fdbfbed399b9a4/libbeat/autodiscover/providers/kubernetes/pod.go#L451